src/uct/api/uct.h

```diff
@@ -1579,6 +1805,25 @@ ucs_status_t uct_iface_reject(uct_iface_h iface,
 ucs_status_t uct_ep_create(const uct_ep_params_t *params, uct_ep_h *ep_p);
 
 
+/**
+ * @ingroup UCT_CLIENT_SERVER
+ * @brief Initiate a synchronized disconnection of an endpoint connected to a
```

Comment:
does `synchronized` means it is blocking call ?

Reply:
@shamisp no, there are callbacks on both sides to notify of the disconnect completion.